extensions/grpc/deployment/src/test/java/io/quarkus/grpc/server/services/BlockingMutinyHelloService.java

```diff
@@ -14,6 +14,10 @@ public class BlockingMutinyHelloService implements Greeter {
     @Override
     @Blocking
     public Uni<HelloReply> sayHello(HelloRequest request) {
+
+        // Force failure if awaiting
+        Uni.createFrom().nullItem().await().indefinitely();
```

Comment:
So @Blocking will timeout this `await::indefinitely` ?

Reply:
No, it will return bill immediately, which is fine.

However, await is forbidden from an event loop thread (it throws an exception), so the call would fail if we call it on an event loop thread.